src/devices/Qmc5883l/RollPointer.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Qmc5883l
+{
+    /// <summary>
+    /// Pointer roll-over function is controlled by ROL_PNT register. When the point roll-over function is enabled, the I2C
+    /// data pointer automatically rolls between 00H ~ 06H, if I2C read begins at any address among 00H~06H.
+    /// ROL_PNT: “0”: Normal, “1”: Enable pointer roll-over function
+    /// </summary>
+    public enum RollPointer : byte
```

Comment:
-I'd suggest to change binary enums into bool unless it's super unobvious what true/false will mean. Also I suggest to use the name `RollI2cPointer`
- what does this mean for user? when should they use it? should this be an implementation detail?